### ISSUE TITLE
[Package Issue]: Cyanfish.NAPS2

### ISSUE THREAD

**Issue by simkoG** [microsoft/winget-pkgs#90081]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

`Cyanfish.NAPS2` install itself without registering the current version number. So when it comes to upgrades winget always tries to upgrade it (with the `--include-unknown` flag). 

Also, the program always opens up after an upgrade.

### Steps to reproduce

```
> winget upgrade

Name                                    Id                               Version        Available     Source
------------------------------------------------------------------------------------------------------------
[list of upgradable packages]
6 upgrades available.
1 package has a version number that cannot be determined. Using "--include-unknown" may show more results.

> winget upgrade --include-unknown

Name                                    Id                               Version        Available     Source
------------------------------------------------------------------------------------------------------------
[list of upgradable packages]
NAPS2 6.1.2                             Cyanfish.NAPS2                   Unknown        6.1.2         winget
7 upgrades available.                                                       ↳ **this is the problem**
```

### Actual behavior

`Cyanfish.NAPS2` always keep updating.

### Expected behavior

`Cyanfish.NAPS` save the current version. 

### Environment

```shell
> winget --info

Windows Package Manager v1.3.2691
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.2251
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2691.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

### Screenshots and Logs

[WinGet-2022-11-30-09-00-30.094.log](https://github.com/microsoft/winget-pkgs/files/10120888/WinGet-2022-11-30-09-00-30.094.log)

**Comment by Trenly** [comment]
This is because the publisher does not write the version to the registry. Since the application doesn't tell Windows what version it is, winget also doesn't know what version it is. In order to fix this, you will need to reach out to the publisher of the application and ask them to set the DisplayVersion registry key when the application is installed.

@msftbot area-external

**Comment by Trenly** [comment]
@msftbot installer-error

**Comment by Trenly** [comment]
Issue is resolved in prerelease versions of NAPS2 - See #92858

**Comment by RokeJulianLockhart** [comment]
> https://github.com/microsoft/winget-pkgs/issues/90081#event-11184392080

Added in https://github.com/microsoft/winget-pkgs/blob/e1a64366cef95013fdaf37676a3cab3573d910fb/manifests/c/Cyanfish/NAPS2/7.1.0/Cyanfish.NAPS2.yaml.
